src/main/java/it/pagopa/transactions/projections/handlers/AuthorizationRequestProjectionHandler.java

```diff
@@ -29,6 +29,9 @@ public Mono<Transaction> handle(AuthorizationRequestData data) {
                 )
                 .flatMap(transactionDocument -> {
                     transactionDocument.setStatus(TransactionStatusDto.AUTHORIZATION_REQUESTED);
+                    String paymentGateway = data.paymentGatewayId();
```

Comment:
How come you define the variable paymentGateway?
```suggestion
                    transactionDocument.setPaymentGateway(data.paymentGatewayId());
```